### ISSUE TITLE
Subchief Arness has got mismatched art (original campaign, chapter 3, Fort Ilkard)

### ISSUE THREAD

**Issue by Ragnarok81** [Finaldeath/overhaul#10]
**Describe the bug**
Uthgard subchief Arness has got mismatched art. His spirte shows a Half-Orc, his portrait art shows an Orc or Half-Orc, but his decapitated head shows a blonde human. 

**Screenshots**
Arness' sprite and description/portrait 
<a href="https://ibb.co/Lvkwg7z"><img src="https://i.ibb.co/Lvkwg7z/20230911141810-1.jpg" alt="20230911141810-1" border="0" /></a>

Arness' decapitated head not matching with the rest of his depiction/art at all
<a href="https://ibb.co/ypgm3yt"><img src="https://i.ibb.co/ypgm3yt/20230911141912-1.jpg" alt="20230911141912-1" border="0" /></a>

**Comment by Finaldeath** [comment]
Aren't that tribe mostly made of half-orcs? I wonder if it's just easier to change the item to use this icon instead?

![iit_midmisc_049](https://github.com/Finaldeath/overhaul/assets/6137144/b43459a4-99a4-4b56-b2b5-1162dedf005b)

**Comment by Ragnarok81** [comment]
@Finaldeath 
Not to my knowledge. Uthgardt barbarians are strong, tall, black haired and blue eyed humans according to the Forgotten Relams Lore:

> The Uthgardt is a collective term used to refer to various human barbarian tribes who worship the deity Uthgar. Each tribe was named after a beast totem of Uthgar who mediated between the deity and his people.
The Uthgardt often had black hair and blue eyes, were predominantly larger than most other humans [...]

https://forgottenrealms.fandom.com/wiki/Uthgardt

**Comment by Finaldeath** [comment]
Yes possibly Bioware overdid it, especially since the description refers to a "man". What area was the encounter in?

Also for reference you can post screenshots directly into a post it'll upload fine usually.

**Comment by Ragnarok81** [comment]
Thanks for the advice, didn't know that.
